mantidproject/mantid
# Minor changes to the QE coverage planning tool

Fixes #15039 

The Indirect and Direct planning tool required to be moved to Interface->Direct category and Interface->Indirect category. However it is mentioned in the issue that this is not possible as it cannot be added to both categories.
The interface now includes an Emin text field for direct, similar to Emax text-field in indirect tab.
The numbers in calcQE have been defined as mass of the neutron, Plank's constant, etc. 

_Most of these implementations were made by @mducle_

**To test:**

`Interface -> Utility -> QECoverage`

Emin text-field with default value of `-10`
When plot is clicked, check that the graph scale readjust itself and `y-axis` starts from `-10`
- Would be good idea to try this out with few negative values and see if interface handles it well

When on Direct Tab `HYSPEC` is selected,
- The default value of `s2` is set to `30` always

Check that other functionality has not been effected on the interface.

@AndreiSavici there have been some calculations updates made by @mducle, maybe it would be a good idea if you could go over them and see if they are working as they should.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

refs #15039
